src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1673,6 +1673,7 @@ FullWidthRepeat:
             Dim IntegerLiteralStart As Integer
             Dim UnderscoreInWrongPlace As Boolean
```

Comment:
Consider assigning `False` here since the value is not explicitly initialized any more. Also, there are several cases below where the value is assigned `UnderscoreInWrongPlace = UnderscoreInWrongPlace Or (Peek(Here - 1) = "_"c)` where `UnderscoreInWrongPlace` is initially `False`. (In short, the `UnderscoreInWrongPlace Or` is unnecessary.)